md-setup/src/main/java/com/wipro/ats/bdre/md/setup/SetupDB.java

```diff
@@ -122,7 +120,8 @@ public static String replaceVars(String line) throws Exception {
     private String[] getColumns(String line) throws Exception {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Define and throw a dedicated exception instead of using a generic one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS00112)